spec/index.html

```diff
@@ -903,12 +902,24 @@ <h3>Algorithm</h3>
               <a>blank node identifier</a> <var>identifier</var>
               in the <a>blank node to quads map</a>,
               creating a new entry if necessary.
-              <div class="issue" data-number="15">
-                It seems that <var>Q</var> must be normalized,
-                so that literals with different syntactic representations
-                but the same semantic representations are merged,
-                and that two graphs differing in the syntactic representation
-                of a literal will produce the same set of blank node identifiers.</div>
+              <details>
+                <summary>Explanation</summary>
+                <p>This establishes the <a>blank node to quads map</a>,
+                  relating each <a>blank node</a> with the set of <a>quads</a>
+                  of which it is a component.</p>
+                <p class="note">
+                  <a data-cite="RDF11-CONCEPTS#dfn-literal">Literal</a> components of
+                  <a>quads</a> are not subject to any normalization.
+                  As noted in
+                  <a data-cite="RDF11-CONCEPTS#section-Graph-Literal">Section 3.3</a>
+                  of [[RDF11-CONCEPTS]],
+                  <a data-cite="RDF11-CONCEPTS#dfn-literal-term-equality">literal term equality</a>
+                  is based on the
+                  <a data-cite="RDF11-CONCEPTS#dfn-lexical-form">lexical form</a>,
+                  rather than the <a data-cite="RDF11-CONCEPTS#dfn-literal-value">literal value</a>,
+                  so two literals `01` and `1` are treated as distinct resources.
```

Comment:
It might be over-writing, but it seems to me that adding datatype makes the explanation more rigorous:
```suggestion
                  so two literals `"01"^^xs:integer` and `"1"^^xs:integer` are treated as distinct resources.
```